lang/src/org/partiql/lang/SqlException.kt

```diff
@@ -33,20 +34,19 @@ import org.partiql.lang.errors.UNKNOWN
  *
  * @param message the message for this exception
  * @param errorCode the error code for this exception
- * @param propertyValueMap context for this error
+ * @param errorContext context for this error, includes details like line & character offsets, among others.
+ * @param internal True to indicate that this exception a bug in ParitQL itself.  False to indicate it was caused by
+ * incorrect usage of APIs or invalid end-user queries.
  * @param cause for this exception
- *
- * @constructor a custom error [message], the [errorCode], error context as a [propertyValueMap] and optional [cause] creates an
- * [SqlException]. This is the constructor for the second configuration explained above.
- *
  */
 open class SqlException(
     override var message: String,
     val errorCode: ErrorCode,
-    val errorContext: PropertyValueMap? = null,
+    errorContextArg: PropertyValueMap? = null,
```

Comment:
It's nullable because it always has been nullable.  Changing this to an argument and defaulting `errorContext` to an empty map was a way of simplifying error handling code that reads context properties without changing all of the call sites (a few  direct, but many indirect) to conform it's new constructor signature, which would have resulted in many more noisy diffs in this series of PRs.  If you want, I can undo this change in another commit against this PR, but I would consider making this argument non-nullable to be out of scope of this work.

Reply:
Ok got it. I don't think it needs to be addressed in this PR. Could you make an issue and link here regarding making `errorContextArg` non-nullable?